.github/workflows/build-and-test.yml

```diff
@@ -0,0 +1,27 @@
+name: build-and-test
+
+on:
+  push:
+    # all branches
+  pull_request:
+    branches:
+      - main
+      - master
+
+  # This enables the Run Workflow button on the Actions tab.
+  workflow_dispatch:
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    steps:
+    
+      - uses: actions/checkout@v2
+
+      - uses: dylan-lang/install-opendylan@v1
+
+      - name: Build binary-data-tests
+        run: ./dylan-compiler -build binary-data-test
+
+      - name: Run binary-data-tests-tests
+        run: _build/bin/binary-data-test
```

Comment:
I guess the names should match the binary, i.e. "test", but it doesn't matter much.

Reply:
Ok. Changed, but maybe the .exe should be named binary-data-tests.